apps/asap-server/src/controllers/teams.ts

```diff
@@ -33,8 +33,7 @@ function transformTeam(team: CMSTeam, members: TeamMember[]): TeamResponse {
     projectSummary: team.data.projectSummary?.iv,
     skills: team.data.skills?.iv || [],
     lastModifiedDate: team.lastModified,
-    pointOfContact: members.find(({ role }) => role === 'Project Manager')
-      ?.email,
+    pointOfContact: members.filter(({ role }) => role === 'Project Manager')[0],
```

Comment:
```suggestion
    pointOfContact: members.find(({ role }) => role === 'Project Manager'),
```
Adding the suggestion to help you, in case you want to accept 🤷🏻‍♂️